src/manifest.json

```
@@ -11,6 +11,11 @@
     "en": "SpreadSheet Editor for kintone"
   },
   "icon": "image/icon.png",
+  "homepage_url": {
+    "ja":"https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/spreadsheet-plugin/",
+    "en":"https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/spreadsheet-plugin/",
+    "zh":"https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/spreadsheet-plugin/"
```

```suggestion
    "en":"https://cybozu.dev/ja/kintone/tips/development/plugins/sample-plugin/spreadsheet-plugin/"
```

他の項目には `zh` を設定していないので不要そうです。
ちなみにこの場合、`homepage_url.en` が使われるようです。